config/core/test-configs.yaml

```diff
@@ -1728,6 +1735,16 @@ test_configs:
     test_plans:
       - baseline
 
+  - device_type: asus-C436FA-Flip-hatch
+    test_plans:
+      - baseline
+      - baseline-nfs
+      - kselftest-filesystems
```

Comment:
Please set the list of test plans as previously discussed, it should be a mix of kselftest and LTP.

Reply:
tested OK on staging:
https://staging.kernelci.org/test/plan/id/60f7e176e297418721c28cf1/

Reply:
> Please set the list of test plans as previously discussed, it should be a mix of kselftest and LTP.

That is to say:

* kselftest-filesystems
* kselftest-futex
* ltp-ipc
* ltp-mm
* ltp-timers

I'm working on the documentation to have that listed on https://kernelci.org/docs.